src/Event/After.php

```
@@ -89,4 +96,60 @@ public function setResult($result)
 
         return $this;
     }
+
+    /**
+     * Get the passed arguments.
+     *
+     * @return array method arguments
+     */
+    public function getArguments()
+    {
+        return $this->arguments;
+    }
+
+    /**
+     * Get an argument by key.
+     *
+     * @param string $key     argument key
+     * @param mixed  $default default return value
+     *
+     * @return mixed
+     */
+    public function getArgument($key, $default = null)
+    {
+        if (! array_key_exists($key, $this->arguments)) {
+            return $default;
```

Instead of a default value, you think it would be better to throw an exception ?